docker/llm/finetune/qlora/cpu/docker/Dockerfile

```diff
@@ -1,19 +1,43 @@
-FROM intel/oneapi-basekit:2023.2.1-devel-ubuntu22.04
+FROM ubuntu:20.04 as key-getter
+ARG http_proxy
+ARG https_proxy
+
+RUN apt-get update && \
+    apt-get install -y curl gpg && \
+    curl -fsSL https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB | gpg --dearmor | tee /root/intel-oneapi-archive-keyring.gpg
+
+FROM mpioperator/intel as builder
+
 ARG http_proxy
 ARG https_proxy
 ENV TZ=Asia/Shanghai
 ARG PIP_NO_CACHE_DIR=false
 ENV TRANSFORMERS_COMMIT_ID=95fe0f5
 
-# retrive oneapi repo public key
-RUN curl -fsSL https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB | gpg --dearmor | tee /usr/share/keyrings/intel-oneapi-archive-keyring.gpg && \
-    echo "deb [signed-by=/usr/share/keyrings/intel-oneapi-archive-keyring.gpg] https://apt.repos.intel.com/oneapi all main " > /etc/apt/sources.list.d/oneAPI.list
+# add public key
+COPY --from=key-getter /root/intel-oneapi-archive-keyring.gpg /usr/share/keyrings/intel-oneapi-archive-keyring.gpg
+RUN echo "deb [signed-by=/usr/share/keyrings/intel-oneapi-archive-keyring.gpg] https://apt.repos.intel.com/oneapi all main " > /etc/apt/sources.list.d/oneAPI.list
 
-# update dependencies
-RUN apt-get update && \
-    # install basic dependencies
+
+RUN mkdir -p /bigdl/data && mkdir -p /bigdl/model && \
+# install pytorch 2.1.0
+    apt-get update && \
+    apt-get install -y python3-pip python3.9-dev python3-wheel git software-properties-common && \
+    pip3 install --upgrade pip && \
+    export PIP_DEFAULT_TIMEOUT=100 && \
+    pip install --upgrade torch==2.1.0 --index-url https://download.pytorch.org/whl/cpu && \
+# install CPU bigdl-llm
+    pip3 install --pre --upgrade bigdl-llm[all] -i https://pypi.tuna.tsinghua.edu.cn/simple/ && \
+# install ipex and oneccl
+    pip install intel_extension_for_pytorch==2.0.100 && \
+    pip install oneccl_bind_pt -f https://developer.intel.com/ipex-whl-stable && \
+# install huggingface dependencies
+    pip install transformers==4.34.0 && \
+    pip install peft==0.5.0 datasets && \
```

Comment:
please update it refer to [this pr](https://github.com/intel-analytics/BigDL/pull/9511)

Reply:
done